cnd/src/btsieve/ethereum/mod.rs

```diff
@@ -24,16 +20,119 @@ use std::collections::HashSet;
 type Hash = H256;
 type Block = crate::ethereum::Block<Transaction>;
 
-pub async fn matching_transaction<C>(
+pub const TRANSACTION_STATUS_OK: u32 = 1;
```

Comment:
This is now unused because the trait defines it's own const.

Reply:
Nice, removed.